esx_service/vmdk_ops.py

```diff
@@ -252,31 +253,57 @@ def getVMDKUuid(vmdk_path):
     except:
         return None
 
+# Create and return the devFS path for a VSAN object UUID.
+def get_vsan_devfs_path(uuid):
+
+    logging.debug("Got volume UUID %s", uuid)
+
+    # Objtool creates a link thats usable to 
+    # read write to vsan object.
+    cmd = "{0} {1}".format(OBJ_TOOL_CMD, uuid)
+    rc, out = RunCommand(cmd)
+    if rc == 0 and os.path.isfile(fpath):
+        return "/vmfs/devices/vsan/{0}".format(uuid), True
+    return None, False
+
+# Clean up vsan devfs path
+def cleanup_vsan_devfs_path(devfs_path)
+    try:
+        os.remove(backing)
+        logging.debug("Unlinked %s for flag %s", backing, cleanup_device)
+    except OSError as ex:
+        logging.error("Failed to remove backing device %s",
+                      backing, str(ex))
+    return None
+
+# Returns the flat file for a VMDK.
+def get_backing_flat_file(vmdk_path):
+    return vmdk_path.replace(".vmdk", "-flat.vmdk")
 
 # Return a backing file path for given vmdk path or none
-# if a backing can't be found.
-def getVMDKBacking(vmdk_path):
-    flatBacking = vmdk_path.replace(".vmdk", "-flat.vmdk")
+# if a backing can't be found. Returns True if clean up 
+# is needed. Always call cleanup_backing_device when done.
```

Comment:
nit: "do not forget to call cleanup_backing_device" is probably a better comment.